### PR TITLE
LW-7691 e2e tests on preprod adjustments

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context

- To run e2e tests against networks like preprod/preview/mainnet, we need to adjust the timeouts to be more generous.
- Fix mint test to burn tokens only from the test policyId, to avoid signing issues.
 
# Proposed Solution

Introduced a new env variable `NETWORK_SPEED=slow|fast`. Default is `fast`, for running e2e tests against the local-testnet.

# Important Changes Introduced

### CODE REVIEW

File: packages/e2e/test/wallet/PersonalWallet/mint.test.ts [hunk 1]
```diff
@@ -21,13 +21,16 @@ const logger = createLogger();
 
 describe('PersonalWallet/mint', () => {
   let wallet: BaseWallet;
+  let alicePolicySigner: util.KeyAgentTransactionSigner;
+  let policyScript: Cardano.NativeScript;
 
-  afterAll(() => {
+  afterAll(async () => {
+    await burnTokens({ policySigners: [alicePolicySigner], scripts: [policyScript], wallet });
```
Comment (iccicci) [inline packages/e2e/test/wallet/PersonalWallet/mint.test.ts:28]: Nice change!

**Review by iccicci — APPROVED** [review]
Very very nice! Good job!

**Review by mkazlauskas — APPROVED** [review]
Great work! :rocket: `test` commit type might be more appropriate for b6dc42c9e6f59081e8e00ceb58e4c8e254f650c7

**Review by AngelCastilloB — APPROVED** [review]
LGTM @mirceahasegan 🏄🏼‍♂️

**Pull request merged**